specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/examples/CopyModel.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "modelId": "{modelId}",
+    "body": {},
+    "copyRequest": {
+      "targetResourceId": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CognitiveServices/accounts/{resourceName}",
+      "targetResourceRegion": "westus2",
+      "copyAuthorization": {
+        "modelId": "{modelId}",
+        "accessToken": "{accessToken}",
+        "expirationDateTimeTicks": 637190189980000000
```

Comment:
We model this as an Int32 (seconds since Unix epoch) but this sample value is an Int64

Reply:
done